README.md

```diff
@@ -1,105 +1,64 @@
-# Awesome Celestia
-An Awesome List of Celestia Resources
+Welcome to the NacionCrypto repository! This repository showcases our collective efforts and contributions to Celestia. Our team is dedicated to supporting and enhancing the Celestia ecosystem through various initiatives and community engagements.
```

Comment:
i don't think this should be merged. can you please edit this PR to fit the existing format of the `## Node operator contributions` section? thanks.